common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -29,33 +29,68 @@
  * according to a specific format.
  */
 public class FormatUtils {
+  /**
+   * Parse a list of <code>Objects</code> into a <code>String</code>.
+   *
+   * @param objs a list of Objects to convert to a String
+   * @return comma-separated concatenation of the string representation returned by Object#toString
+   *         of the individual objects
+   */
   public static String parametersToString(Object... objs) {
     StringBuilder sb = new StringBuilder("(");
-    for (int k = 0; k < objs.length; k ++) {
-      if (k != 0) {
-        sb.append(", ");
+    if (objs != null) {
+      for (Object obj : objs) {
+        if (sb.length() != 1) {
```

Comment:
Done. Let me know if you'd prefer to show a something else than an empty space for empty string. As it is, an empty array and an array containing an empty string produce the same output: ().